doc/global.md

```diff
@@ -0,0 +1,43 @@
+## Global
+
+The global functions are defined at the pigpio module level.
+
+#### Functions
+  - [getTick()](#getTick)
+  - [tickDiff(startTick, endTick)](#tickDiff)
```

Comment:
the tickDiff link doesn't work. Replacing `#tickDiff` with `#tickdiffstarttick-endtick` should fix it.

Reply:
Thx, not sure how to test these links and was guessing from other .md files